tests_e2e/orchestrator/lib/agent_test_loader.py

```diff
@@ -148,18 +167,22 @@ def _load_test_suite(description_file: Path) -> TestSuiteInfo:
               - "bvts/vm_access.py"
             images: "endorsed"
             location: "eastuseaup"
-            owns-vm: true
+            owns_vm: true
 
         * name     - A string used to identify the test suite
-        * tests    - A list of the tests in the suite. Each test is specified by the path for its source code relative to
-                     WALinuxAgent/tests_e2e/tests.
+        * tests    - A list of the tests in the suite. Each test can be specified by a string (the path for its source code relative to
+                     WALinuxAgent/tests_e2e/tests), or a dictionary with two items:
+                        * source: the path for its source code relative to WALinuxAgent/tests_e2e/tests
+                        * blocks_suite: [Optional; boolean] If True, a failure on the test will stop execution of the test suite (i.e. the
+                          rest of the tests in the suite will not be executed). By default, a failure on a test does not stop execution of
+                          the test suite.
```

Comment:
Now tests can be described with a simple string (path to their source code), or an object with 2 properties: 'source' and 'blocks_suite'